### PR TITLE
Fix ugly exceptions management

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Because this is completly useless.

### CODE REVIEW

File: Security/Handler/RoleSecurityHandler.php [hunk 1]
```diff
@@ -67,8 +67,6 @@ public function isGranted(AdminInterface $admin, $attributes, $object = null)
                 || $this->authorizationChecker->isGranted($attributes, $object);
         } catch (AuthenticationCredentialsNotFoundException $e) {
             return false;
-        } catch (\Exception $e) {
-            throw $e;
```
Comment (greg0ire) [inline Security/Handler/RoleSecurityHandler.php]: Mother of god…

**Comment by greg0ire** [comment]
Proposed changelog note : "remove brain-damaged crap"

**Comment by Soullivaneuh** [comment]
Wait before merge! I found other crappy catchs...

**Comment by Soullivaneuh** [comment]
Okay, I finished to fix my heart attack.

File: Security/Handler/AclSecurityHandler.php [hunk 2]
```diff
@@ -216,15 +214,10 @@ public function findObjectAcls(\Traversable $oids, array $sids = array())
...
             $acls = $this->aclProvider->findAcls(iterator_to_array($oids), $sids);
-        } catch (\Exception $e) {
-            if ($e instanceof NotAllAclsFoundException) {
-                $acls = $e->getPartialResult();
-            } elseif ($e instanceof AclNotFoundException) {
-                // if only one oid, this error is thrown
-                $acls = new \SplObjectStorage();
-            } else {
-                throw $e;
-            }
```
Comment (greg0ire) [inline Security/Handler/AclSecurityHandler.php]: hurts my brain

**Comment by greg0ire** [comment]
The planet thanks you @Soullivaneuh !

**Pull request merged**